go.mod

```diff
@@ -1,9 +1,9 @@
 module oras.land/oras-go/v2
 
-go 1.23
+go 1.23.7
```

Comment:
This only passes with the patch version update? Seems like we ran into trouble with helm when oras-go did this before.

Reply:
Or we can do:
```go
go 1.23.0

toolchain go1.23.3
```

Could you elaborate the trouble we ran into before?

Reply:
They wouldn't update to oras-go, I think it was 1.2.5, because it required a patch version of go. They finally updated when we hit 1.2.6.

I haven't read up to understand how toolchain works or the effects of it, but that may be better.